cmd/security-profiles-operator/main.go

```diff
@@ -38,8 +41,12 @@ import (
 
 const (
 	jsonFlag               string = "json"
+	seccompFlag            string = "with-seccomp"
```

Comment:
This flag doesn't seem to be used anywhere.

Reply:
Yeah, I think I'll just remove it from now. In fact, I think I might even remove the EnableSeccomp flag and make only SELinux optional until we figure out a nicer way to instantiate the DS based on options.

Reply:
Removed